packages/foreman/nodejs-unidiff/nodejs-unidiff.spec

```diff
@@ -0,0 +1,46 @@
+%global npm_name unidiff
+%global enable_tests 1
+
+Name: nodejs-%{npm_name}
+Version: 0.0.4
+Release: 1%{?dist}
+Summary: diff with unified diff format handling
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/mvoss9000/unidiff#readme
+Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+Requires: npm(diff) >= 2.2.2
+Requires: npm(diff) < 3.0.0
```

Comment:
This package should be bundled if it depends on diff < 3.0.0 because our packaging includes 3.0.1. An alternative is to locally patch the dependency. Like we do here:
https://github.com/theforeman/foreman-packaging/blob/38e9a718b513261ab2f3bb8ed339e1bbe4f2be78/packages/katello/nodejs-react-bootstrap-tooltip-button/nodejs-react-bootstrap-tooltip-button.spec#L27-L29

Reply:
something like?

```
%prep
%setup -q -n package
%nodejs_fixdep diff '> 2.2.2 <= 3.0.1'
```

Reply:
@ekohl updated again - thanks